draft-brand-indicators-for-message-identification-latest.md

```diff
@@ -203,7 +203,14 @@ BIMI removes the substantial burden of curating and maintaining an Indicator dat
 
 The structure of BIMI is as follows:
 
-* Domain Owners: Publish their preferred Brand Indicators via the [DNS].
+* Domain Owners: 
+  * Fully implement the [DMARC] mechanism, to include:
+    * Creating and publishing in [DNS] a [DMARC] policy record that meets the following criteria:
+      * The policy record MUST express a Requested Mail Receiver policy of “quarantine” or “reject” for the Organizational Domain and all sub-domains
+      * The Requested Mail Receiver policy MUST apply to a percentage of mail not less than 100
```

Comment:
I don't think this is quite correct.
The requested policy may be reject, with a policy of 50% such that 50% of mail will be at reject and 50% of mail will be at quarantine.
For BIMI this is fine, and desired as a method of getting folks from quarantine to reject.
We could replace the above 2 points with...

* The policy record MUST express either a Requested Mail Receiver policy of “quarantine” with an effective percentage of 100%, or a Requested Mail Receiver policy of “reject” (with any percentage value).
* Is a subdomain policy is published it MUST NOT be "none"
* Be published for the Organizational Domain, and any subdomains thereof